CHANGELOG.md

```diff
@@ -34,24 +35,92 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - `getSession` now takes a `checkDatabase` parameter instead of using the `access_token_blacklisting` config value 
 - Updated plugin interface version to 2.21
 
+### Configuration Changes
+
+- `access_token_signing_key_dynamic` is now deprecated, only used for requests with CDI<=2.18
+- `access_token_blacklisting` is now deprecated, only used for requests with CDI<=2.18
+- Renamed `access_token_signing_key_update_interval` to `access_token_dynamic_signing_key_update_interval`
+
 ### Database Changes
 
 - Added new `useStaticKey` field into session info
 - Manual migration is also required if `access_token_signing_key_dynamic` was set to true
 
 #### Migration steps for SQL
-- TODO
 - If using `access_token_signing_key_dynamic` false:
-  - `ALTER TABLE table ADD COLUMN use_static_key BOOLEAN NOT NULL DEFAULT(false);`
-  - 
-- If using `access_token_signing_key_dynamic` true:
   - `ALTER TABLE table ADD COLUMN use_static_key BOOLEAN NOT NULL DEFAULT(true);`
-  - `DELETE FROM tbl_OldTableName WHERE id in (SELECT id FROM tbl_NewTableName)` 
+  - ```sql
+    INSERT INTO jwt_signing_keys(key_id, key_string, algorithm, created_at)
+      select CONCAT('s-', created_at_time) as key_id, value as key_string, 'RS256' as algorithm, created_at_time as created_at
+      from session_access_token_signing_keys;
+    ```
+- If using `access_token_signing_key_dynamic` true:
+  - `ALTER TABLE table ADD COLUMN use_static_key BOOLEAN NOT NULL DEFAULT(false);` 
```

Comment:
- replace table with table name?

Have you tried to run these migration queries in postgresql and mysql and they work?

Reply:
After this is fixed, please feel free to merge this PR into the base branch